plugins/attachment-resources/src/components/AttachmentStyleBoxEditor.svelte

```diff
@@ -86,7 +86,13 @@
       haveUnsavedChanges = false
     }
 
-    await descriptionBox.createAttachments()
+    /*
```

Comment:
Why we need this changes in this PR?

Reply:
My bad, I just reproduced this A LOT when tested mention links. And I thought it would be merged soon anough. Will move to a separated PR